src/sentry/workflow_engine/processors/detector.py

```diff
@@ -58,11 +66,34 @@ def process_detectors(
                         "group_key": result.group_key,
                     },
                 )
+                continue
+
+            if result.result is not None:
+                occurrence, status_change = None, None
+                if isinstance(result.result, IssueOccurrence):
+                    occurrence = (
+                        result.result if isinstance(result.result, IssueOccurrence) else None
+                    )
+                    payload_type = PayloadType.OCCURRENCE
+                else:
+                    status_change = result.result
+                    payload_type = PayloadType.STATUS_CHANGE
+
+                produce_occurrence_to_kafka(
```

Comment:
🎉 1/2 of milestone 2. heh.

Reply:
Yeah, there's wasn't much to implementing it so I just went ahead. Most of the work is producing the occurrence anyway.